### PR TITLE
[xml] Update romanian.xml

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
I'm trying again. I hope it's good now. I checked both Croatian and English localizations.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 1]
```diff
@@ -1059,16 +1071,17 @@
                     <Item id="6293" name="Lățime constantă"/>
                     <Item id="6207" name="Semn de carte pentru afișaj"/>
                     <Item id="6223" name="Istoricul schimbărilor afișajului"/>
+                    <Item id="6296" name="Afișează în text"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:1074]: id 6296 is repeated.

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 2]
```diff
@@ -1059,16 +1071,17 @@
...
-                    <Item id="6237" name="Se adaugă marcatorul de coloană indicând poziția acestuia cu un număr zecimal.
-Se pot defini mai mulți marcatori de coloană utilizând spațiul alb pentru a separa diferitele numere."/>
                     <Item id="6231" name="Lățimea chenarului"/>
                     <Item id="6235" name="Fără vârf"/>
                     <Item id="6208" name="Umplutură"/>
                     <Item id="6209" name="Stânga"/>
                     <Item id="6210" name="Dreapta"/>
                     <Item id="6212" name="Fără distracție"/>
+		    <Item id="6295" name="Istoricul schimbărilor"/>
+		    <Item id="6296" name="Afișează în text"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:1084]: id 6296 is repeated.

**Review by donho — CHANGES_REQUESTED** [review]
remove redundant entry

File: PowerEditor/installer/nativeLang/romanian.xml (hunk 2 above)
Comment (AndreiMiloiu) [reply PowerEditor/installer/nativeLang/romanian.xml:1083]: I solved it. I removed the repetead entry.

File: PowerEditor/installer/nativeLang/romanian.xml [hunk 3]
```diff
@@ -973,51 +973,63 @@
...
                     <Item id="6227" name="Limitare linie"/>
                     <Item id="6228" name="Implicit"/>
                     <Item id="6229" name="Aliniere"/>
                     <Item id="6230" name="Indentare"/>
-                    <Item id="6206" name="Afișare numere linii"/>
+                    <Item id="6206" name="Afișare numere de linii"/>
                     <Item id="6207" name="Afișare semne de carte"/>
+		    <Item id="6295" name="Istoricul schimbărilor"/>
+		    <Item id="6223" name="Afișare în marjă"/>
+		    <Item id="6296" name="Afișare în text"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/romanian.xml:994]: These 3 entries should not be here

**Review by donho — CHANGES_REQUESTED** [review]
Please follow the suggestion.

File: PowerEditor/installer/nativeLang/romanian.xml (hunk 3 above)
Comment (AndreiMiloiu) [reply PowerEditor/installer/nativeLang/romanian.xml]: I removed them.

**Review by donho — APPROVED** [review]

**Comment by AndreiMiloiu** [comment]
Great! Thank you @donho for patience and for help!

**Pull request closed without merging**